<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlDataDialog</class>
 <widget class="QDialog" name="controlDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>930</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control Data Specifications</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>661</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Control data sets are used to confirm operation of the analyzer by running it on data whose structure is already known. Chose among the following sets to create the control set. Complex data sets can be created by combining multiple sets together by using the Add button. Click Finish when the control data set is complete.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="dataTypeGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>677</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>140</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select whether you are creating a data set showing a signal or masking a region&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="title">
    <string>Type of Data Set</string>
   </property>
   <widget class="QRadioButton" name="dataButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Input Signal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrumental Filter Effects</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="weightsButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Input Mask</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="noiseButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrumental Noise Effects</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="beamButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrumental Beam Effects</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="representationGroup">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>270</y>
     <width>677</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>90</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the angular resolution in degrees for the control set. This establishes the underlying grid used to distribute the data points on the spherical surface.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="title">
    <string>Data Representation</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>651</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="representationVLayout">
     <item>
      <layout class="QHBoxLayout" name="resolutionLayout">
       <item>
        <widget class="QLabel" name="decResolutionLabel">
         <property name="text">
          <string>Dec/Latitude Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="decResolutionValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RAResolutionLabel">
         <property name="text">
          <string>RA/Longitude Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RAResolutionValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="coordinateGroup">
       <property name="title">
        <string>Coordinate System</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>40</y>
          <width>621</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="coordinateLayout">
         <item>
          <widget class="QRadioButton" name="RADecButton">
           <property name="text">
            <string>RA/Dec</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="galLatLongButton">
           <property name="text">
            <string>Galactic Latitude/Longitude</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="earthLatLongButton">
           <property name="text">
            <string>Terrestrial Latitude/Longitude</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="dataSetFrame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>500</y>
     <width>677</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>220</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="dataSetGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the type of data set you want to add to the control data set&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>Available Data Sets</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>161</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="dataSetLayout">
      <item>
       <widget class="QRadioButton" name="uniformButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Uniform Sky</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="regionalButton">
        <property name="text">
         <string>Uniform &amp;Region</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="deltaButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Delta Function</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gaussianButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Gaussian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="checkerButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Checkerboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="harmonicButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Specific Harmonic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="supplementalGroup">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Supplemental Information</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>441</width>
       <height>36</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="sigLabel">
        <property name="text">
         <string>Signal Strength</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sigValue">
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sigUnit">
        <property name="text">
         <string>(arbitrary units)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="blankPage"/>
    <widget class="QWidget" name="regionPage">
     <widget class="QGroupBox" name="regionGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>471</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Restrict data to the region bounded by</string>
      </property>
      <widget class="QWidget" name="layoutWidget_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>451</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="regionLayout">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="regionDecLabel">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Dec/Latitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="3">
         <widget class="QLabel" name="regionRALabel">
          <property name="text">
           <string>RA/Longitude</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="regionTopLabel">
          <property name="text">
           <string>Top</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLabel" name="regionFromLabel">
          <property name="text">
           <string>From</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="regionFromValue">
          <property name="text">
           <string>-180.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="regionBottomLabel">
          <property name="text">
           <string>Bottom</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="regionBottomValue">
          <property name="text">
           <string>-90.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="regionToLabel">
          <property name="text">
           <string>To</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="regionTopValue">
          <property name="text">
           <string>90.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLineEdit" name="regionToValue">
          <property name="text">
           <string>180.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="deltaPage">
     <widget class="QGroupBox" name="peakGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>471</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Peak Location (deg)</string>
      </property>
      <widget class="QWidget" name="layoutWidget_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>451</width>
         <height>76</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="peakLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="peakDecLabel">
          <property name="text">
           <string>Dec/Latitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="decValue">
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="peakRALabel">
          <property name="text">
           <string>RA/Longitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="RAValue">
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="fwhmLabel">
          <property name="text">
           <string>Std Dev Width</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="fwhmValue">
          <property name="text">
           <string>0.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="fwhmUnit">
          <property name="text">
           <string>(deg)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="checkerboardPage">
     <widget class="QGroupBox" name="checkerGroup">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Chekcerboard Pattern (deg)</string>
      </property>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>386</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="checkerLayout">
        <item>
         <widget class="QLabel" name="decScaleLabel">
          <property name="text">
           <string>Dec/Latitude Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="decScaleValue">
          <property name="text">
           <string>0.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="RAScaleLabel">
          <property name="text">
           <string>RA/Longitude Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RAScaleValue">
          <property name="text">
           <string>0.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SHPage">
     <widget class="QGroupBox" name="SHGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>471</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Spherical Harmonic Values</string>
      </property>
      <widget class="QWidget" name="layoutWidget_13">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>30</y>
         <width>451</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SHLayout">
        <item>
         <widget class="QLabel" name="lLabel">
          <property name="text">
           <string>l Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lSpinner"/>
        </item>
        <item>
         <spacer name="SHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="mLabel">
          <property name="text">
           <string>m Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mSpinner"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="opGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>780</y>
     <width>677</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>45</height>
    </size>
   </property>
   <property name="title">
    <string>Operation</string>
   </property>
   <widget class="QRadioButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="subtractButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtract</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="replaceButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="invertButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mirrorButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Mirror</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>890</y>
     <width>671</width>
     <height>36</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
